### ISSUE TITLE
Adding ability to define custom static tags (in configuration)

### ISSUE THREAD

**Issue by zeugfr** [poxet/Influx-Capacitor#9]
It would be nice to add in the configuration (globally or at counter level) the ability to add custom tags (with static key+value) that would be added to all metrics sent to Influx.

For example, on all servers I would like to get a tag called "Project" and, on each machine, I would put the name of the related project.
That would allow me to query easily all performance counters of all machines for a specified project:
```
select min(value) 
from disk 
where time > now() - 1h 
and category='"LogicalDisk"' and counter='"% Idle Time"' 
and project='Gamma'
group by hostname,instance
```
Thanks !

**Comment by poxet** [comment]
Great idea and should be easy to add!

**Comment by poxet** [comment]
This feature is added to the project and will be included from version 1.0.8.13.
